CMakeLists.txt

```diff
@@ -137,7 +137,7 @@ ENDIF()
 IF ("${CMAKE_BUILD_TYPE}" MATCHES "Debug")
     ADD_DEFINITIONS(-DDEBUG=1 -UNDEBUG)
     IF (NOT WINDOWS)
-        ADD_DEFINITIONS(-Wall -Werror -Wextra -Wno-unused-parameter)
```

Comment:
Why was this pulled out?

Reply:
Ah, sorry about that.

When building Alembic against some older versions of HDF5, debug builds would fail due to warnings in HDF5Public.h.

Removing it was a convenience on my local branch that I should have excluded from the branch I pushed with the other changes. 

I'll put it back in shortly!